_data/retail.yml

```
@@ -373,19 +448,25 @@ websites:
     email_address: customerservice@sierra.com
     twitter: sierraofficial
     facebook: sierraofficial
+    regions:
+      - us
 
   - name: Snapdeal
     url: https://www.snapdeal.com/
     img: snapdeal.png
     twitter: SnapdealTech
     facebook: Snapdeal
     email_address: help@snapdeal.com
+    regions:
+      - in
 
   - name: speedcube.com.au
     url: https://www.speedcube.com.au/
     img: speedcubecomau.png
     twitter: speedcubecomau
     facebook: speedcubecomau
+    regions:
+      - au
```

```suggestion
    facebook: speedcubecomau
```
Speedcube [ships internationally](https://www.speedcube.com.au/pages/shipping) and bills in 6 currencies.